molgenis-data/src/main/java/org/molgenis/data/DataService.java

```diff
@@ -218,28 +218,32 @@
   <E extends Entity> E findOne(String entityTypeId, Query<E> q, Class<E> clazz);
 
   /**
-   * Adds an entity to it's repository
+   * Adds an entity to it's repository Note: when the entityType contains one-to-many attributes,
```

Comment:
Better?

Note: the caller is responsible for updating the other side of bidirectional relationships (e.g. one-to-many attributes).